import-service/serverless.yml

```diff
@@ -0,0 +1,53 @@
+service: import-service
+
+provider:
+  name: aws
+  runtime: nodejs16.x
+  
+  stage: dev
+  region: eu-west-1
+
+  iamRoleStatements:
+    - Effect: "Allow"
+      Action: "s3:ListBucket"
+      Resource:
+        - "arn:aws:s3:::s3-serverless-integration"
+    - Effect: "Allow"
+      Action: 
+        - "s3:*"
```

Comment:
- s3:PutObject
- s3:PutObjectAcl
Would be enough to make it work rather then allow all actions